docs/pages/querying_existing_db/getting_started/index.rst

```diff
@@ -16,23 +16,27 @@ To use the AstroDB Toolkit with Python, you need the database
 located on your machine.
 That likely entails cloning the database's repository or downloading
 the database files directly.
+
 Once you have the database files, you can access them using the
-`astrodb-utils` package.
+`astrodb-utils` package. To load the database, the relevant functions
+are in the :py:mod:`loaders<astrodb_utils.loaders>` module.
+These functions use the database settings file (TOML format) to
+create an SQLite database file and load the database into Python.
 
 .. code-block:: python
 
-    from astrodb-utils import load_astrodb
+    from astrodb-utils import load_db_from_json
 
     # Load the database into a variable called `db`
     # and make a sqlite file
-    db = load_astrodb("path/to/database.sqlite")
+    db = load_db_from_json("path/to/database.toml")
```

Comment:
May be clearer if this is:
```bash
db = load_db_from_json(settings_file="path/to/database.toml")
```
That is more explicit as to what is being passed and has a clearer parameter name.